core/rfb.js

```diff
@@ -53,6 +53,7 @@ export default class RFB extends EventTargetMixin {
         this._shared = 'shared' in options ? !!options.shared : true;
         this._repeaterID = options.repeaterID || '';
         this._showDotCursor = options.showDotCursor || false;
+        this._wsProtocols = options.wsProtocols || ['binary'];
```

Comment:
Argh. Are we still including that `"binary"` default. That is very non-standard.

@samhed, could you remove that and just have nothing as the default once this is merged?

Reply:
Sure

Reply:
Fixed in e14aa4d0fe43a939378a3dd7666806fa7018dfe7